tests/unit/pages/quiz-attempt/QuizAttempts.spec.jsx

```diff
@@ -56,10 +85,12 @@ describe('QuizPage for student', () => {
         new RegExp(URLs.finishedQuizzes.getById.replace(':id', mockQuizId))
       )
       .reply(200, mockQuiz)
+
+    mockAxiosClient.onGet('/finished-quizzes//').reply(200, mockFinishedQuizzes)
```

Comment:
It will make this mock more understandable.

```suggestion
    mockAxiosClient.onGet(URLs.finishedQuizzes.getById.replace(':id','')).reply(200, mockFinishedQuizzes)
```
But the better way to handle this is to mock `useParams` with `mockQuizId` as you need.